hugegraph-core/src/main/java/com/baidu/hugegraph/structure/HugeElement.java

```diff
@@ -73,7 +73,7 @@
     private boolean defaultValueUpdated;
 
     public HugeElement(final HugeGraph graph) {
-        E.checkArgument(graph != null, "HugeElement graph can't be null");
+//        E.checkArgument(graph != null, "HugeElement graph can't be null");
```

Comment:
Why remove check

Reply:
There is an operator sink scenario where graph would be null